dotCMS/src/main/java/com/dotcms/publisher/pusher/PushUtils.java

```diff
@@ -75,10 +75,10 @@ private static void addFilesToCompression(TarArchiveOutputStream taos, File file
 	    			taos.putArchiveEntry(new TarArchiveEntry(file, dir + "/" + file.getName()));
 				if (file.isFile()) {
 			        // Add the file to the archive
-					BufferedInputStream bis = new BufferedInputStream(new FileInputStream(file));
-					IOUtils.copy(new FileInputStream(file), taos);
-					taos.closeArchiveEntry();
-					bis.close();
+					try(BufferedInputStream bis = new BufferedInputStream(new FileInputStream(file))) {
```

Comment:
Why would we not do a 
`Files.newInputStream(file.toPath()` knowing the Memory and GC issues with FileInputStreams (even when they are closed properly)